### PR TITLE
i18n(fr): Add `integrations-guide/alpinejs` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
i18n(fr): Add `integrations-guide/alpinejs` in French language

#### Related issues & labels (optional)

- Linked to https://github.com/withastro/docs/issues/6052
- Suggested label: i18n

### CODE REVIEW

**Comment by McFlyPartages** [comment]
LGTM 😜

File: src/content/docs/fr/guides/integrations-guide/alpinejs.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+L'intégration d'Alpine.js ne prend pas en charge les configurations personnalisées pour le moment.
+
+## Exemples
+
+* L'exemple [Astro Alpine.js](https://github.com/withastro/astro/tree/latest/examples/framework-alpine) montre comment utiliser Alpine.js dans un projet Astro.
+
+[astro-integration] : /fr/guides/integrations-guide/
+
+[astro-ui-frameworks] : /fr/core-concepts/framework-components/#using-framework-components
```
Comment (VoxelMC) [inline src/content/docs/fr/guides/integrations-guide/alpinejs.mdx:129]: These lines seem out of place - are they intended to be here still?

**Review by VoxelMC — COMMENTED** [review]
The translation is wonderful! Thank you Thomas!

I made one comment on some errant lines. Once resolved, I can say:
`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!` (I _do_ have some working knowledge (B1+) in French, if that helps~)

File: src/content/docs/fr/guides/integrations-guide/alpinejs.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/alpinejs.mdx:129]: Hello! This is normal, in the English version, It's written the same way

File: src/content/docs/fr/guides/integrations-guide/alpinejs.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+title: '@astrojs/alpinejs'
+description: Apprenez à utiliser l'intégration du framework @astrojs/alpinejs pour étendre le support des composants dans votre projet Astro.
+githubIntegrationURL: 'https://github.com/withastro/astro/tree/main/packages/integrations/alpinejs/'
+category: renderer
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Cette **[intégration Astro][astro-integration]** ajoute [Alpine.js](https://alpinejs.dev/) à votre projet afin que vous puissiez utiliser Alpine.js n'importe où sur votre page.
+## Installation
```
Comment (pioupia) [inline src/content/docs/fr/guides/integrations-guide/alpinejs.mdx:12]: Need space just before this line (this don't affect the final render of the page)
```suggestion

## Installation
```

**Review by pioupia — APPROVED** [review]
Everything's perfect, just missing two line breaks if you want to avoid an offset in the file (as @VoxelMC said), but that doesn't affect the final rendering.

File: src/content/docs/fr/guides/integrations-guide/alpinejs.mdx (hunk 1 above)
Comment (Yan-Thomas) [reply src/content/docs/fr/guides/integrations-guide/alpinejs.mdx:131]: Yeah, this is normal Markdown syntax for "link shortcuts", just a note these can't have a space between the square brackets and `:`, so it just gets rendered as text instead.

File: src/content/docs/fr/guides/integrations-guide/alpinejs.mdx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+L'intégration d'Alpine.js ne prend pas en charge les configurations personnalisées pour le moment.
+
+## Exemples
+
+* L'exemple [Astro Alpine.js](https://github.com/withastro/astro/tree/latest/examples/framework-alpine) montre comment utiliser Alpine.js dans un projet Astro.
+
+[astro-integration] : /fr/guides/integrations-guide/
+
+[astro-ui-frameworks] : /fr/core-concepts/framework-components/#using-framework-components
```
Comment (Yan-Thomas) [inline src/content/docs/fr/guides/integrations-guide/alpinejs.mdx:131]:
```suggestion
[astro-integration]: /fr/guides/integrations-guide/

[astro-ui-frameworks]: /fr/core-concepts/framework-components/#using-framework-components
```

Comment (thomasbnt) [reply src/content/docs/fr/guides/integrations-guide/alpinejs.mdx]: Thanks!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🚀

File: src/content/docs/fr/guides/integrations-guide/alpinejs.mdx (hunk 1 above)
Comment (VoxelMC) [reply src/content/docs/fr/guides/integrations-guide/alpinejs.mdx]: Ah, so it was the spaces then. Thanks Yan!

**Pull request merged**